sds_data_manager/lambda_code/SDSCode/instruments/l1b_codice.py

```diff
@@ -24,15 +24,16 @@ def lambda_handler(event: dict, context):
 
     # Get the environment variables
     bucket = os.environ["S3_BUCKET"]
-    prefix = os.environ["S3_KEY_PATH"]
+    prefixes = json.loads(os.environ["S3_KEY_PATH"])
 
     # Retrieves objects in the S3 bucket under the given prefix
-    try:
-        s3 = boto3.client("s3")
-        object_list = s3.list_objects_v2(Bucket=bucket, Prefix=prefix)["Contents"]
-        logger.info(f"Object list: {object_list}")
-    except KeyError:
-        logger.warning("No files present.")
+    for prefix in prefixes:
+        try:
+            s3 = boto3.client("s3")
+            object_list = s3.list_objects_v2(Bucket=bucket, Prefix=prefix)["Contents"]
+            logger.info(f"Object list: {object_list}")
+        except KeyError:
```

Comment:
Can we make the logger description more descriptive? (i.e. "missing the following files: ...") If this is temporary and planned to be removed, then it's fine as-is

Reply:
The lambas may be changed entirely. I'll put a todo in case we want to eventually keep this to have a more descriptive logger.